src/utils/vscodeUtils.ts

```diff
@@ -101,3 +104,22 @@ export function getNodeEditorLabel(node: IAzureNode): string {
 function isAccountTreeItem(treeItem: IAzureTreeItem): boolean {
     return (treeItem instanceof MongoAccountTreeItem) || (treeItem instanceof DocDBAccountTreeItemBase);
 }
+
+export function getDocumentTreeItemLabel(document: IMongoDocument | RetrievedDocument): string {
+    for (let field of documentDefaultFields) {
+        if (document.hasOwnProperty(field)) {
+            if (document[field]) { //ignore if false-y value: null, undefined, "".
```

Comment:
let value = document[field];
if (value) ...

Slightly better perf, but  more importantly, less chance of mistakes or botched bug fixes.